assessment.md

```diff
@@ -2,32 +2,125 @@
+  flags
+  and commands have been incorporated.
+  * :sunglasses: = Programs are able to run with `pyenv` and `pipenv` with additional
+  flags
+  and commands to improve testing while also providing assistance to other students
+  in their use of `pyenv` and `pipenv`.
 * Linting Python programs with `pylint` and `flake8`
+  * :disappointed: = `pylint` and `flake8` not installed or setup.
+  * :confused: = `pylint` and `flake8` installed but incorrectly configured and
+  unable to run.
+  * :neutral_face: = `pylint` and `flake8` able to lint programs but program returns
+  linting errors.
+  * :smiley: = `pyling` and `flake8` able to lint programs successfully, programs
+  contain no linting errors.
+  * :sunglasses: = `pylint` and `flake8` able to lint programs, programs contain
+  no linting errors, troubleshooting assistance provided for installation and configuring
+  of `pylint` and `flake8` to other students.
 * Formatting Python programs with `black`
+  * :disappointed: = `black` not installed, setup, or used.
+  * :sunglasses: = Ran `black` successfully and file was formatted.
 * Basic testing of Python programs with `pytest`
-* Advanced testing of Python programs with `pytest`
+  * :disappointed: = Did not use `pytest`.
+  * :confused: = `pytest` employed, but with inadequate test cases.
+  * :neutral_face: = Few test cases were created.
+  * :smiley: = Thorough, adequate test cases.
+  * :sunglasses: = Thorough, adequate, and documented test cases.
 * Calculating code coverage of a `pytest` test suite
+  * :disappointed: = Did not check code coverage
+  * :sunglasses: = Checked code coverage with `pytest`
 * Reporting code coverage through an online provider
+  * :disappointed: = Did not check coverage
+  * :neutral_face: = Checked code coverage using an online provider
+  * :sunglasses: = Checked code coverage through an online provider and
+  documentation provided.
 * Using docstrings to document a Python program
+  * :disappointed: = Did not use docstrings
+  * :confused: = Incorrectly use of docstrings which are unidentifiable to the
+  compiler.
+  * :neutral_face: = Added docstrings, but they do not fully describe the
+  functions'/methods' effect as a command.
+  * :smiley: = Successfully added docstrings with useful description, but they
+  lack some proper syntax.
+  * :sunglasses: = Successfully added docstring with useful description and
+  proper syntax.
 * Adopting and applying appropriate naming conventions
+  * :disappointed: Used unconventional naming styles foreign to python's syntax
+  * :confused: Used appropriate naming styles that are not entirely conventional
+  * :neutral_face: Used inappropriate naming styles that are not prescriptive or
+  descriptive
+  * :smiley: Used appropriate naming styles, but is missing syntax used with
+  variables, function, and methods
+  * :sunglasses: Used appropriate naming styles with correct syntax used with
+  variables, functions, and methods
```

Comment:
Equal sign that was being used above after the emoji name is now gone...make sure you follow the same format throughout the whole document